validation/mesoscale_turbulence/immersed_coarse_baroclinic_adjustment.jl

```diff
@@ -85,12 +85,8 @@ function run_simulation(closure, grid)
     #####
 
     simulation = Simulation(model; Δt, stop_time)
-
-    wall_clock = Ref(time_ns())
-
-    add_callback!(simulation, progress, IterationInterval(10))
-
-    suffix = closure isa AdvectiveSkewClosure ? "advective" : "diffusive"
+    add_callback!(simulation, progress, IterationInterval(144))
+    suffix = closure isa AdvectiveFormulation ? "advective" : "diffusive"
```

Comment:
It seems that this line of code does not work as expected: it returns "diffusive" in both cases. How to fix this ?

Reply:
```suggestion
    suffix = closure isa SkewAdvectionISSD ? "advective" : "diffusive"
```
but I think you need to import it:
```
using Oceananigans.TurbulenceClosures: SkewAdvectionISSD
```

Reply:
I will try.

Reply:
It works ! Thanks @simone-silvestri  I will push this little change.